pkg/apis/servicecatalog/types.go

```diff
@@ -277,9 +277,9 @@ type ClusterServiceClassSpec struct {
 
 // ServiceClassStatus represents status information about a ServiceClass.
 type ServiceClassStatus struct {
-	// PresentInCatalog indicates that the ServiceClass is present in the
-	// broker's catalog.
-	PresentInCatalog bool
+	// MissingFromBrokerCatalog indicates that the broker removed the service from its
```

Comment:
Instead of Missing, wonder if Removed would be a better term (especially since that term is used in the comment for this field). Thoughts?

Reply:
or: NotPresentInCatalog?

Reply:
Yeah, Removed is probably better.

It's not missing, I know exactly where it went!

Reply:
I wonder whether we will correctly handle existing instances if the plan appears again in the catalog after it was removed?